airflow/bin/cli.py

```diff
@@ -467,7 +467,7 @@ def serve_logs(filename):  # noqa
 
 def worker(args):
     env = os.environ.copy()
-    env['AIRFLOW_HOME'] = settings.AIRFLOW_HOME
+    env['AIRFLOW_HOME'] = conf.get_airflow_home()
```

Comment:
Use the configuration convenience method that evaluates the current `AIRFLOW_HOME` rather than the one that's evaluated once when `settings.py` gets loaded.